src/System.Collections/src/System.Collections.builds

```diff
@@ -4,21 +4,8 @@
   <ItemGroup>
     <Project Include="System.Collections.csproj" />
     <Project Include="System.Collections.csproj">
-      <TargetGroup>netstandard1.3</TargetGroup>
```

Comment:
> I'm still not sure if we want to remove this

You and @weshaggard need to agree here and then tell me what to do ;)  I can either merge this or remove this last commit.